draft-add-requirements.md

```diff
@@ -55,66 +92,119 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 document are to be interpreted as described in BCP 14 {{RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
-# Discovery of associated resolvers {#first-usecase}
-
+{: #first-usecase}
+# Discovery of associated resolvers
 A client may begin with information from the attached networks, and/or resolvers known from configuration.
 This information may be used to then discover one or more associated or equivalent encrypted resolvers.
 These are expected to be operated by the same entity that provides the resolver the client started with, but
-may not have identical behaviour. For example one entity may offer a set of encrypted resolvers with different levels of
-filtering or proximity.
+may not have identical behaviour. For example one entity may offer a set of encrypted resolvers with different levels
```

Comment:
I don't the example, what do you mean by "different levels of filtering" ?

Reply:
An example is Cloudflare. Choose between unfiltered (1.1.1.1), malware filtered (1.1.1.2), or malware and adult content filtered (1.1.1.3).

Reply:
Got it, adding an example will be helpful.

Reply:
Added.